src/components/shared/Logo.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+const Logo: React.FC<{}> = () => {
+  const logoClass = 'text-right  text-lowercase logo-message';
```

Comment:
if we use bootstrap styles, we can use them directly without additional variables.

Reply:
ok, thanks